CONTRIBUTING.md

```diff
@@ -30,6 +30,8 @@ $ rubocop -V
 * Write [good commit messages][3].
 * Use the same coding conventions as the rest of the project.
 * Commit and push until you are happy with your contribution.
+* If you are fixing a bug, prefix the commit message with `[Fix #github-issue-number]`.
+* If it is only a documentation change, use `[ci skip]` so that the [CI won't run][8].
```

Comment:
Rails people follow it so I thought this might make sense in the contributing guide. Removed now.